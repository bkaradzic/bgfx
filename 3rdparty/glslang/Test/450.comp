#version 450 core
