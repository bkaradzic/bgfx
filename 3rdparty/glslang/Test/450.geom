#version 450 core

in gl_PerVertex {
    float gl_CullDistance[3];
} gl_in[];

out gl_PerVertex {
    float gl_CullDistance[3];
};

void main()
{
    gl_CullDistance[2] = gl_in[1].gl_CullDistance[2];
}
