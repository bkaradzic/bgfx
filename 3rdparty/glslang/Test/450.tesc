#version 450 core

in gl_PerVertex {
    float gl_CullDistance[3];
} gl_in[gl_MaxPatchVertices];

out gl_PerVertex {
    float gl_CullDistance[3];
} gl_out[4];

void main()
{
    gl_out[gl_InvocationID].gl_CullDistance[2] = gl_in[1].gl_CullDistance[2];
}
