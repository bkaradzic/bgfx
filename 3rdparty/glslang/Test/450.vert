#version 450 core

out gl_PerVertex {
    float gl_CullDistance[3];
};

void main()
{
    gl_CullDistance[2] = 4.5;
}

out bool outb;         // ERROR
out sampler2D outo;    // ERROR
out float outa[4];
out float outaa[4][2];
struct S { float f; };
out S outs;
out S[4] outasa;
out S outsa[4];
struct SA { float f[4]; };
out SA outSA;
struct SS { float f; S s; };
out SS outSS;
