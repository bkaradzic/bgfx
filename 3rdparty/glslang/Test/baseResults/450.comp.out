450.comp
Warning, version 450 is not yet complete; most version-specific features are present, but some are missing.

Shader version: 450
local_size = (1, 1, 1)
0:? Sequence
0:?   Linker Objects


Linked compute stage:

ERROR: Linking compute stage: Missing entry point: Each stage requires one entry point

Shader version: 450
local_size = (1, 1, 1)
0:? Sequence
0:?   Linker Objects

